webapp/home/subdomains/genome/content/assembly.py

```diff
@@ -31,6 +31,17 @@
         ],
         "button_link": "https://usegalaxy.org.au/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fbgruening%2Fflye%2Fflye",
     },
+    {
+        "title_html": '<code>Unicycler</code> - assembly with Illumina, PacBio or Nanopore data - bacteria only',
+        "description_html": """
+            <p>
+              Hybrid assembly pipeline for bacterial genomes, uses both Illumina reads and long reads (PacBio or Nanopore).
+              </p>""",
+        "inputs": [
+            {'datatypes': ['fastq']},
+        ],
+        "button_link": "https://usegalaxy.org.au/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Funicycler%2Funicycler%2F0.5.0%2Bgalaxy1",
```

Comment:
Oh, if you delete the `%2F0.5.0%2Bgalaxy1` from the end of the tool URL it will default to the latest version, so we don't need to update it :)

Reply:
done, thanks!